examples/cli/Cargo.toml

```diff
@@ -2,35 +2,49 @@
 default-run = "xmtp_cli"
 edition = "2021"
 keywords = ["xmtp", "messaging", "web3", "group-chat"]
+license.workspace = true
 name = "xmtp_cli"
 readme = "README.md"
 repository = "https://github.com/xmtp/libxmtp"
 version.workspace = true
-license.workspace = true
 
 [[bin]]
 name = "xmtp_cli"
 path = "cli-client.rs"
 
 [dependencies]
-tracing = { workspace = true, features = ["valuable"] }
-tracing-subscriber = { workspace = true, features = ["json", "valuable", "env-filter", "ansi", "chrono"] }
-valuable = { version = "0.1", features = ["derive"] }
-valuable-serde = "0.1"
+chrono.workspace = true
 clap = { version = "4.4.6", features = ["derive"] }
+color-eyre = "0.6"
 ethers.workspace = true
 futures.workspace = true
 hex.workspace = true
+kv-log-macro = "1.0.7"
+log = { version = "0.4", features = [
```

Comment:
can remove this, i made everything use `tracing` to be consistent and preserve the `async` context of spans & added K/V pairs to `xmtp_mls`. I made sure the log format was the same. JSON logs can be passed into third party log viewer.

You can use k/v pairs with `=` before the log message, i.e

`info!(key = value, "my message {}", format_arg);`